app/controllers/account/billing-info/invoices/list.js

```diff
@@ -0,0 +1,193 @@
+import Controller from '@ember/controller';
+import { computed } from '@ember/object';
+import EmberTableControllerMixin from 'open-event-frontend/mixins/ember-table-controller';
+
+export default class extends Controller.extend(EmberTableControllerMixin) {
+
+  @computed()
+  get upcomingInvoiceColumns() {
+    return [
+      {
+        name      : 'Invoice ID',
+        valuePath : 'invoiceId'
+      },
+      {
+        name      : 'Event Name',
+        valuePath : 'eventName'
+      },
+      {
+        name      : 'Date Issued',
+        valuePath : 'dateIssued'
+      },
+      {
+        name      : 'Outstanding Amount',
+        valuePath : 'amount'
+      },
+      {
+        name      : 'View Invoice',
+        valuePath : 'invoiceLink'
+      }
+
+    ];
+  }
+
+  @computed()
+  get AllColumns() {
```

Comment:
These are arrays with plain objects. Can be a normal property instead of a computed property